vtex-apps/checkout-shipping
# CHK-470: Display place details based on the type of location form

#### What problem is this solving?

In order to improve the UX, when NewAddressForm is using the autocomplete input instead of postal code input, it will display a different PlaceDetails visualisation. For Brasil, instead of CEP, it will display the street name, number and neighborhood.

Depends on https://github.com/vtex-apps/place-components/pull/41

#### How should this be manually tested?

- [workspace](https://geolocation--checkoutio.myvtex.com/cart/add?sku=289)
- select delivery options
- insert "Praia de botafogo, 200" and click on the first option
- it should display "Rua Praia de Botafogo, 200 - Botafogo"

#### Screenshots or example usage

Before | After
-|-
![image](https://user-images.githubusercontent.com/26108090/105362543-670b0500-5bd1-11eb-9aff-12bd91805c6f.png) | ![image](https://user-images.githubusercontent.com/26108090/105363252-32e41400-5bd2-11eb-946b-9427e15971be.png)

(it does look weird, but that's a problem with PlaceDetails that will be addressed on the future)
